src/third-party-tags/inizio.ts

```diff
@@ -1,19 +1,20 @@
 import type { GetThirdPartyTag } from '../types';
 
-const onLoad = () => {
-	const handleQuerySurveyDone = (
-		surveyAvailable: boolean,
-		survey: { measurementId: string },
-	) => {
-		if (surveyAvailable) {
-			if (window.googletag) {
-				window.googletag.cmd.push(() => {
-					window.googletag?.pubads().setTargeting('inizio', 't');
-				});
-			}
-			console.log(`surveyAvailable: ${survey.measurementId}`);
+export const handleQuerySurveyDone = (
```

Comment:
The convention for testing is to export an object called `_` (underscore) at the end of the file, like so:

```ts
export const _ = {
  handleQuerySurveyDone,
  onLoad,
}
```

See some examples here:
- https://github.com/guardian/frontend/blob/40c9d66bf4832b9aef7ca389db535b5e282f9185/static/src/javascripts/projects/common/modules/commercial/geo-utils.js#L26
- https://github.com/guardian/consent-management-platform/blob/2edc662cf92880cd38ad0e1d02e5a997baddf8df/src/onConsentChange.ts#L60

Reply:
Great, good to know!